src/components/tables/PrefixAlerts/index.tsx

```diff
@@ -44,14 +45,23 @@ function PrefixAlertTable() {
         setColumnToSort,
     } = useTableState(TablePrefixes.prefixAlerts, 'catalog_prefix', 'asc');
 
+    const selectedTenant = useTenantStore((state) => state.selectedTenant);
+
     const query = useMemo(() => {
-        return getNotificationSubscriptionsForTable(pagination, searchQuery, [
-            {
-                col: columnToSort,
-                direction: sortDirection,
-            },
-        ]);
-    }, [columnToSort, pagination, searchQuery, sortDirection]);
+        return selectedTenant
```

Comment:
Please add a quick comment about why we're returning null since the `null` getting handled is a little "deep" in the store.